src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,198 @@
+6. Sign in to your wallet account if you haven't already, then confirm the connection and budget amount. Below is an example screen.<br /><img src="" alt="Example budget confirmation screen with accept and decline buttons" style="max-width:400px" class="img-outline" />When the connection is successful, your browser will show a success screen.<br /><img src="" alt="Web page displaying a connection successful screen" style="max-width:400px" class="img-outline"/>
+</Steps> 
+
+## Adjust your budget 
+
+<Steps>
+1. Click the cog in the upper-right of the extension.
+    <br /><img src="" alt="Extension showing the cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
+2. Select the *Budget* tab.
+    <br /><img src="" alt="Budget tab with remaining balance and option to change budget amount" style="max-width:300px" class="img-outline"/>
+3. Enter the new amount in the **Budget amount** field.
+4. Select the **Monthly** toggle to enable/disable monthly renewals of your budget.
+5. Click **Submit changes**. A new tab opens to your wallet provider. 
+6. Sign in to your account if you haven't already, then confirm the new budget amount.
+</Steps>
+
+## Change your continuous payment settings
+
+A continuous payment is a payment that recurs at your set rate for as long as you are active on a web monetized page. When you leave the page, the payments stop.
+
+Continuous payments are enabled by default and sent at a rate of $0.60 USD per hour. Use the instructions below to change the rate or disable continuous payments. 
+
+<Steps>
+1. Click the cog in the upper-right of the extension.
+    <br /><img src="" alt="Extension showing the cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
+2. Select the *Rate* tab.
+    <br /><img
+    src=""
+    alt="Rate tab with rate of pay field and continuous payment toggle"
+    style="max-width:300px"
+    class="img-outline"
+    />
+3. Enter the amount the extension will send, per hour, to web monetized sites.
+4. Select the **Continuous payment** toggle to enable/disable continuous payments. This is a global setting that applies to all web monetized pages you visit.
+</Steps>
+
+## Make a one-time payment 
+
+You can make a one-time payment (OTP) to a page instead of, or in addition to, continuous payments. The amount of the OTP is deducted from your budget. 
+
+<Steps>
+1. Ensure the URL shown above the amount field is the page you intend to pay. 
+    <br /><img
+    src=""
+    alt="Extension showing an available balance of five dollars, with one dollar in the amount field and send now button available"
+    style="max-width:300px"
+    class="img-outline"
+    />
+2. Enter the amount and click **Send now**. The button's text will briefly change to *Payment successful*, then back to *Send now*.
```

Comment:
The button text doesn't change anymore. We show the success/failure response above the form now.
It'll clear the form and disable the submit button until new amount is entered (button text will continue to say "Send now" - but a spinner will be shown while payment goes through).